kansha/notifications.py

```diff
@@ -14,14 +14,15 @@
 import json
```

Comment:
We can't have such a module knowing all the history messages. Each component must handle its own messages